recipes/floris/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "FLORIS" %}
+{% set version = "1.1.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/NREL/FLORIS/archive/v{{ version }}.tar.gz
+  sha256: 982ffe5d091cd1af7e100935acfb6305f799b99936af3f10f4f9763c1186a0b5
+  # Download the tar-ball and do `openssl sha256 file.tar.gz` to update
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+
+  run:
+    - h5py
+    - matplotlib
```

See https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib
```suggestion
    - matplotlib-base
```